community/rfcs/GenAIExamples-24-05-16-001-Using_MicroService_to_implement_ChatQnA.md

```diff
@@ -0,0 +1,166 @@
+# Author
+liang1.lv@intel.com, feng.tian@intel.com, haihao.shen@intel.com, sihan.chen@intel.com, letong.han@intel.com 
+
+# Status
+Under Review
+
+# Objective
+This RFC aims to solve the problem of designing a flexible architecture for RAG (Retrieval-Augmented Generation) application by adopting a microservice architecture. The goal is to enable easier deployment where one or multiple microservices can compose a megaservice. Each megaservice interfaces with a gateway, allowing users to access services through endpoints exposed by the gateway.
+
+# Motivation
+The problem of designing a flexible architecture for RAG is valuable to solve because it allows for more scalable and maintainable deployment of RAG systems. While related work in microservice architectures exists, the specific requirements and design considerations for RAG systems may necessitate a tailored approach.
```

Comment:
The architecture is general and RAG is the first example that we want to apply, so please clarify this.